external/js/kyber/src/curve/nist/point.ts

```diff
@@ -99,11 +99,13 @@ export default class NistPoint implements Point {
             }
 
             const xRed = x.toRed(this.ref.curve.curve.red);
+            // @ts-ignore
             const aX = xRed.redMul(new BN(this.ref.curve.curve.a));
             // y^2 = x^3 + ax + b
             const y2 = xRed.redSqr()
                 .redMul(xRed)
                 .redAdd(aX)
+                // @ts-ignore
```

Comment:
Do you really need thoses? I've removed it and `npm run build` seems to work.

Reply:
It's mostly if I directly depend on the cothority-sources for development... And my dev-environment is stricter than the cothority-environment.